CHANGELOG.md

```diff
@@ -4,6 +4,151 @@ The format is based on [Keep a Changelog].
 
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
+## [v0.17.0] - 2023-04-14
+
+This is a significant release and the major breaking changes to be aware of are:
+
+### Server backpressure
+
+This release changes the server to be "backpressured" and it mostly concerns subscriptions.
+New APIs has been introduced because that and the API `pipe_from_stream` has been removed.
+
+Before it was possible to do:
+
+```rust
+	module
+		.register_subscription("sub", "s", "unsub", |_, sink, _| async move {
+			let stream = stream_of_integers();
+
+			tokio::spawn(async move {
+				sink.pipe_from_stream(stream)
+			});
+		})
+		.unwrap();
+```
+
+After this release one must do something like:
+
+```rust
+	module
+		.register_subscription("sub", "s", "unsub", |_, pending, _| async move {
+			pending.accept().await?;
+
+			let stream = stream_of_integers();
+
+			loop {
+				tokio::select! {
+					_ = sink.closed() => break Ok(()),
+					maybe_item = stream.next() => {
+						let Some(item) = maybe_item else {
+							break Ok(())
+						};
+						let msg = SubscriptionMessage::from_json(&item)?;
+						if let Err(e) = sink.send_timeout(msg) {
+							match e {
+								// The subscription is closed.
+								Err(TrySendError::Closed(_)) => break Ok(()),
+								// Channel is full, just drop the subscription
+								Err(TrySendError::Full(_)) => break Ok(()),
+							}
+						}
+					}
+				}
+			}
+		})
+		.unwrap();
```

Comment:
Maybe worth just pointing to some example where we've used this (I think we used it in a few places) too because they also show that one can write some helper function to do it if needed in multiple places :)